syntax = "proto3";

package PbInfluxLogger;

message Configuration
{
    string  mqtt_host     = 1;
    uint32  mqtt_port     = 2;
    string  vehicle_id    = 3;
    string  device_id     = 4;
    string  influx_host   = 5;
    uint32  influx_port   = 6;
    bool    influx_https  = 7;
    string  influx_bucket = 8;
    string  influx_orgid  = 9;
    string  influx_token  = 10;
    repeated string networks = 11;
}